src/main/resources/com/google/api/codegen/java/disco_schema.snip

```diff
@@ -0,0 +1,141 @@
+@extends "java/common.snip"
+@extends "java/method_sample.snip"
+
+@snippet generate(apiFile)
```

Comment:
In terms of naming/concepts, I want to decouple the input/transformer from the output/view model+templates. What that means concretely is that the transformers are discovery-specific, but the view model and templates shouldn't be discovery specific, so they shouldn't have 'disco' in names or in comments. This file in particular could probably be named `message.snip`, because it's generating a message class. 

The motivating reason for this is to support other input formats in the future, but which use the same generated message templates.